editor/editor_spin_slider.cpp

```diff
@@ -136,8 +137,10 @@ void EditorSpinSlider::_gui_input(const Ref<InputEvent> &p_event) {
 	}
 
 	Ref<InputEventKey> k = p_event;
-	if (k.is_valid() && k->is_pressed() && k->is_action("ui_accept")) {
-		_focus_entered();
+	if (k.is_valid() && k->is_pressed()) {
+		if (k->is_action("ui_accept")) {
+			_focus_entered();
+		}
 	}
```

Comment:
how come this was split into a nested if?

Reply:
That was a remnant from an experiment I forgot to fix. I've pushed a reversion. Thanks!